frontend/src/LearnMoreContent.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+const LearnMoreContent = () => {
+  return (
+    <div>
+      <h3>Understanding Classroom Availability: Rules and Guidelines</h3>
+      <p>
+        1. Timetable Accuracy: Empty classrooms are based on the official
+        timetable provided by the institution's ERP system. Professors might
```

Comment:
This feels like it is written by ChatGPT

Reply:
Yes! I have taken help from ChatGPT for the content. But I have checked multiple times, the content generated by ChatGPT is good to go. Kindly, check from you end also.